peribolos/knative-sandbox.yaml

```diff
@@ -803,14 +803,14 @@ orgs:
           reconciler-test: write
         teams:
           API Core WG Leads:
-            description: The Working Group leads for Serving API Core
+            description: The Working Group leads for Serving
             members:
             - dprotaso
             privacy: closed
```

Comment:
I'm confused why we have both `api core wg leads` and `serving wg leads`

Is this an intermediate state until we get all the repos switched over to the new name?

Reply:
Yup - exactly